### DISCUSSION TITLE
Validation Summary of Error Messages in a Modal popup

### DISCUSSION THREAD

**Discussion by vbav** [react-hook-form/react-hook-form#3423] [Q&A]
@bluebill1049 - We spoke yesterday regarding an IE 11 issue. Have another question. I have a huge form with 20 fields. Is there a way we can show all the error messages in a Modal popup. I know we have the errors object but how do we get to show them in a Modal window rather than showing inline?

**Comment by bluebill1049** [comment]
pass down the props into the Modal component?

**Comment by vbav** [comment]
@bluebill1049 - I will try today and confirm. Thanks.
